experimental/lib/Transforms/LSQSizing/LSQSizingSupport.cpp

```diff
@@ -559,3 +571,12 @@ int CFDFCGraph::getEarliestStartTime(mlir::Operation *op) {
              ? 0
              : nodes.at(opName).earliestStartTime;
 }
+
+bool CFDFCGraph::isConnectedToLSQ(mlir::Operation *op) {
+  for (mlir::Operation *destOp : op->getUsers()) {
+    if (isa<handshake::LSQOp>(destOp)) {
+      return true;
+    }
+  }
```

Comment:
To be safe, use the following to check if the load/store is connected to an LSQ:
https://github.com/EPFL-LAP/dynamatic/blob/f258670bbefca8fdaa12c18bff2c19ca2f93dfdc/lib/Support/TimingModels.cpp#L135-L137